common/build.gradle

```diff
@@ -52,12 +57,14 @@ android {
         debug {
             testCoverageEnabled enableCodeCoverage
             minifyEnabled false
+            buildConfigField("boolean", "SHOULD_TRUST_DEBUG_BROKERS", "$shouldTrustDebugBrokers")
         }
         release {
             minifyEnabled false
             debuggable false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             consumerProguardFiles 'consumer-rules.pro'
+            buildConfigField("boolean", "SHOULD_TRUST_DEBUG_BROKERS", "$shouldTrustDebugBrokers")
```

Comment:
No need to set this twice (in both debug and release). Just set this once in the `defaultConfig`